jax/numpy/lax_numpy.py

```diff
@@ -1240,6 +1240,26 @@ def select(condlist, choicelist, default=0):
   return output
 
 
+@_wraps(onp.bincount, lax_description="""\
+Jax adds the optional `length` parameter which specifies the output length, and
+defaults to ``x.max() + 1``. It must be specified for bincount to be compilable.
+""")
+def bincount(x, weights=None, minlength=0, *, length=None):
+  if length is None:
+    length = x.max() + 1
+  length = _max(length, minlength)
+  x = array(x)
+  if ndim(x) != 1:
+    raise ValueError("only 1-dimensional input supported.")
+  if weights is None:
+    weights = array(1, dtype='int32')
```

Comment:
In general elsewhere in this file I've used the `int32` scalar type. I prefer to avoid stringly-typed code where possible. And you save two whole characters here because the `int32` name is in the current namespace!

Reply:
Done! Two characters saved!